Rogue-Ape-Studios/Crusader
# 🍱 Implement Wave System Changes in `Temple`

## Content

Applied the changes mentioned in #333 to the `Temple` level.

## Changes


### Updated
`Assets/Scenes/Levels/Temple.unity` : Was updated / renamed. You can add describe the changes you made here.

## Testing

The feature / Bugfix can be testing by following these steps:

1. Start level.
2. Notice infinite gameplay.

Aids #322
